go-controller/pkg/cni/helper_linux.go

```diff
@@ -103,6 +104,28 @@ func moveIfToNetns(ifname string, netns ns.NetNS) error {
 	return nil
 }
 
+func saveVFRepName(cid, repName string) error {
```

Comment:
should this be saved in json format so if later other information need to cached then extending this would be easier ? (e.g to restore VF name as well)

Reply:
I can add the -default suffix, how can we tell it's a default or additional when multiple networks are supported?

Reply:
Done